code/Drops/CrateDrops.cs

```diff
@@ -13,18 +12,35 @@ public static class CrateDrops
 	private static readonly List<float> _cumulativeDropPercentagesTools = new();
 	private static bool _init;
 
+	static CrateDrops()
+	{
+		Initialize();

```

Comment:
`Initialize()` this is called in three other places, I would assume we only need to call it here now, right?

Reply:
I kept them airing on the side of caution because of s&box hot-reloading. I'll throw caution to the wind and test it

Reply:
![image](https://user-images.githubusercontent.com/66768086/227365101-0dcf2a19-18ff-4ac8-8f53-1db120652a99.png)
Yeah. S&box cries about it. I'm not sure if it actually impacts anything but I don't want anyone to have to deal with the annoying popup every hotload.